docs/src/man/comparisons.md

```diff
@@ -1,77 +1,173 @@
+| Reduce multiple values   | `df['z'].mean(skipna = False)`                        | `combine(df, :z => mean)`                                     |
+|                          | `df['z'].mean()`                                      | `combine(df, :z => mean ∘ skipmissing => :z_mean)`            |
+| Add new columns          | `df.assign(x_mean = df['x'].mean())`                  | `transform(df, :x => mean => :x_mean)`                        |
+| Rename columns           | `df.rename(columns = {'x': 'x_new'})`                 | `rename(df, :x => :x_new)`                                    |
+| Pick & transform columns | `df.assign(x_mean = df['x'].mean())[['x_mean', 'y']]` | `select(df, :x => mean, :y)`                                  |
+| Sort rows                | `df.sort_values(by = ['x'])`                          | `sort(df, :x)`                                                |
+
+Note that Julia propagates `missing` data by default for safety reasons. The `skipmissing` function
+can be used to remove missing data. See more details at the [Additional Differences](@ref) section below.
+
+### Grouping data and aggregation
+
+| Operation                    | pandas                                                                                    | DataFrames.jl                               |
+|:-----------------------------|:------------------------------------------------------------------------------------------|:--------------------------------------------|
+| Aggregate by groups          | `df.groupby('grp')['x'].mean().reset_index()`                                             | `combine(groupby(df, :grp), :x => mean)`    |
+| Aggregate and add columns    | `df.join(df.groupby('grp')['x'].mean(), on='grp', rsuffix='_mean')`                       | `transform(groupby(df, :grp), :x => mean)`  |
+| Aggregate and select columns | `df.join(df.groupby('grp')['x'].mean(), on='grp', rsuffix='_mean')[['grp','x_mean','y']]` | `select(groupby(df, :grp), :x => mean, :y)` |
+
+### More advanced commands
+
+| Operation                 | pandas                                                                       | DataFrames.jl                                                 |
+|:--------------------------|:-----------------------------------------------------------------------------|:--------------------------------------------------------------|
+| Complex Function          | `df[['z']].agg(lambda v: np.mean(np.cos(v)))`                                | `combine(df, :z => (v -> mean(cos, skipmissing(v))))`         |
+|                           | `df['z'].mean()`                                                             | `combine(df, :z => (v -> mean(skipmissing(v))))` |
+| Transform several columns | `df.agg({'x': max, 'y': min})`                                               | `combine(df, :x => maximum, :y => minimum)`                   |
```

Comment:
the result of the operation is different here

Reply:
the same in the next example

Reply:
Are you OK with writing examples that does not exercise DataFrames features? Doing `mean(df.z)` is just a regular Julia function. I guess that's fine.... just pointing out the difference because pandas impose the same syntax with any number of columns as noted in the above comment.

Reply:
But `df['z'].mean()` seems to me just a numpy functionality:
```
>>> np.array([1,2,3]).mean()
2.0
```
I think in this table we should focus on showing how functions from DataFrames.jl work (otherwise there is no precise borderline where to stop discussing functionalities of Base and standard library).

Reply:
> But df['z'].mean() seems to me just a numpy functionality

That's true but pandas is giving its users an easier way to write the code. Similarly, Julia Base/Statistics is giving the user an easier way for us to write `mean(df.z)`.

Reply:
I'd say it's fine to document some essential features that are not strictly speaking provided by pandas, as long as it's only a few cases. Also, we could add a link to https://docs.julialang.org/en/v1/manual/noteworthy-differences/ to the Python and R sections.

Reply:
OK, I think it's best to just remove this particular example because this section should demonstrate how to work with complex functions, and this is definitely not going to be one. We already have another complex function example so that would be fine. Feel free to bring up more thoughts/suggestions.